vnext/Microsoft.ReactNative/Views/FlyoutViewManager.cpp

```diff
@@ -284,8 +287,14 @@ void FlyoutShadowNode::updateProperties(const folly::dynamic &&props) {
         m_isLightDismissEnabled = true;
       if (m_isOpen) {
         auto popup = GetFlyoutParentPopup();
-        if (popup != nullptr)
+        if (popup != nullptr) {
           popup.IsLightDismissEnabled(m_isLightDismissEnabled);
+        }
+      }
+      if (m_xamlRootChangedRevoker && !m_isLightDismissEnabled) {
+        m_xamlRootChangedRevoker.revoke();
+      } else if (m_isLightDismissEnabled) {
```

Comment:
>else if [](start = 8, length = 7)

(nit) maybe switch if and else if for readability?
if (m_isLightDismissEnabled)
else if(m_xamlRootChangedRevoker)

Reply:
thanks for taking a look! :) I took this out in the new commit.